ViewPumpSample/src/main/java/io/github/inflationx/viewpump/sample/MyViewInterceptor.java

```diff
@@ -0,0 +1,38 @@
+package io.github.inflationx.viewpump.sample;
+
+import android.content.Context;
+import android.support.annotation.Nullable;
+import android.util.AttributeSet;
+import android.view.View;
+
+import io.github.inflationx.viewpump.InflateRequest;
+import io.github.inflationx.viewpump.InflateResult;
+import io.github.inflationx.viewpump.Interceptor;
+
+public class MyViewInterceptor implements Interceptor {
```

Comment:
I'm still struggling to understand the flow of the interceptors, I feel like the intercept method should have a default that should shouldn't have to override.

Maybe we provide a `SimpleInteceptor` where `intercept` is populated and we just for the user to impliment the `inflateView` part.